groot/routes.py

```diff
@@ -1,17 +1,52 @@
 from flask import Flask, Response, render_template, request, redirect, url_for, flash
 from run import app, db
-
+from groot.models import *
+import hashlib
 
 
 @app.route("/")
 @app.route('/login', methods=['POST', 'GET'])
 def login():
-    return render_template('login.html')
-        
+    if request.method == 'POST':
+        try:
+            email = request.form.get('email')
+            password = request.form.get('password')    
+            user = User.query.filter_by(email=email).first()
+            if user:
+                hash_pw = hashlib.sha1(password.encode('utf-8')).hexdigest().upper()
```

Comment:
SHA is ok, but I suggest taking a look at other encryption, like bcrypt, which can be implemented very easily with flask. just a recommendation though.

Reply:
First i tried to work with bcrypt but it brought me errors that i couldn't managed to solve, so i tried SHA and it worked fine for me.